src/queryClient.ts

```diff
@@ -109,5 +124,6 @@ export default (config: Partial<QueryClientConfig>) => {
     Hydrate,
     mutations,
     axios,
+    focusManager,
```

Comment:
Would it make sens to also return the queryConfig here ? It might be useful for consumers.